gen/dcos-config.yaml

```diff
@@ -1317,11 +1317,8 @@ package:
       MESOS_WORK_DIR={{ mesos_master_work_dir }}
       MESOS_ZK=zk://zk-1.zk:2181,zk-2.zk:2181,zk-3.zk:2181,zk-4.zk:2181,zk-5.zk:2181/mesos
       SASL_PATH=/opt/mesosphere/lib/sasl2
-{% switch has_mesos_max_completed_tasks_per_framework %}
-{% case "true" %}
+      MESOS_MAX_COMPLETED_FRAMEWORKS={{ mesos_max_completed_frameworks }}
```

Comment:
Why was this `switch` removed?

Reply:
I thought since we have added a default value to `mesos_max_completed_frameworks `, the switch is no longer needed as it will always be true.

Reply:
I see. Can a user set the value to empty in ocnfig.yaml? If that's not possible, then can we get rid of `has_mesos_max_completed_frameworks ` and `has_mesos_max_completed_tasks_per_framework ` variables altogether if they are always going to be set?

Reply:
I think it was possible. But that feels like a validation error. Removed the `has_ ` variables and make sure a value always present.